packages/noports_core/lib/src/sshnp/sshnp_impl.dart

```diff
@@ -114,6 +110,13 @@ abstract class SSHNPImpl implements SSHNP {
   static String getNamespace(String device) => '$device.sshnp';
   String get namespace => getNamespace(params.device);
 
+  // ====================================================================
+  // Auxiliary
+  // ====================================================================
+
+  @protected
+  AtSSHKeyUtil get keyUtil;
```

Comment:
Only the base class for SSHKeyUtil is exposed, with the intention that it is overridden with the specific type needed based on the implementation.